inc/related-posts.php

```diff
@@ -42,6 +42,11 @@ public static function admin_enqueue_scripts()
             
         if (isset($_GET['page']) && $_GET['page'] == 'st_related_posts') {
             wp_enqueue_style('st-taxonomies-css');
+
+            //localize scripts
+            wp_localize_script( 'st-admin-js', 'taxopress_relatedposts_localize',[
```

Comment:
@Deji98 I told you to always check existing code. Why do we need to localize st-admin-js twice? That'll affect one and thereby causing others from working.

Go back to where we enqueue the admin js script and update existing localize with new data.

All these little things you're overlooking has complications on customer websites.

You should not only be after having a code that works but also considering the effect and how they integrate with the whole plugin.

When working with existing feature, only add new code if it doesn't exist before, otherwise, update old code with your changes.

You can see in the admin.js that other strings are localized and that should tell you need to update it and not add new one.